src/Topologies/topology2d.jl

```diff
@@ -132,3 +132,65 @@ boundary_tag(topology::Topology2D, boundary_name::Symbol) =
     findfirst(==(boundary_name), boundary_names(topology))
 
 boundary_faces(topology::Topology2D, boundary) = topology.boundaries[boundary]
+
+function neighboring_elements(topology::Topology2D, elem)
+
+    # Each elem in a Topology2D will have at most 8 neighboring elements
+    #
+    #    o------o------o------o
+    #    |      |      |      |
+    #    |      |      |      |
+    #    o------o------o------o
+    #    |      |      |      |
+    #    |      |      |      |
+    #    o------o------o------o
+    #    |      |      |      |
+    #    |      |      |      |
+    #    o------o------o------o
+    #
+    #
+    # Except for corner elements of the cube (they will have 7)
+    #
+    #    o------o------o------o
+    #    |      |      |      |
+    #    |      |      |      |
+    #    o------o------o------o
+    #    |      |      |      |
+    #    |      |      |      |
+    #    o------o------o------o
+    #    |      |      |   ^
+    #    |      |      | < these two faces are the same at a cube corner
+    #    o------o------o
+
+    neigh_elems = []
+    op_faces = []
```

Comment:
Best to make these typed:
```suggestion
    neigh_elems = Int[]
    op_faces = Int[]
```